### DISCUSSION TITLE
What am I doing wrong here?

### DISCUSSION THREAD

**Discussion by rennwaffen** [Staubgeborener/klipper-backup#135] [Q&A]
![image](https://github.com/user-attachments/assets/6a5a7391-0ed4-4d74-8828-5b33a05ba2ef)

I've reinstalled and regenerated the token. I've added more files and removed filter extensions. Env below:

github_token=ghp_[redacted]
github_username=[redacted]
github_repository=[redacted]
branch_name="main"
commit_username="[redacted]"
commit_email="[redacted]@[redacted]-[redacted]"

backupPaths=( \
"mainsail/*" \
"printer_data/config/*" \
)

exclude=( \
"*.swp" \
"*.tmp" \
)

**Comment by Staubgeborener** [comment]
1. Please create a issue with the "Create issue from discussion" button on the right side
2. run `~/klipper-backup/script.sh --fix` and try again
3. if the issue still appears, please run the script with `~/klipper-backup/script.sh --debug` and show the whole output

Btw, is `~/printer_data/config/mainsail.cfg` a symbolic link like the output explains? Please check via `ls -l ~/printer_data/config/mainsail.cfg` and see [this](https://github.com/Staubgeborener/Klipper-Backup/issues/69#issuecomment-1965839873) post about backing up symbolic links.

**Reply by Staubgeborener** [reply]
I am closing this as there is no feedback
